lib/helper.js

```diff
@@ -146,9 +148,43 @@ const getIfExists = get => {
         retryInterval,
         retryTimeout,
       );
-      return await get(tag);
+      const nodeIds = await get(tag);
+      function getText() {
+        return this.innerText;
+      }
+      let elementWrappers = nodeIds.map(nodeId => {
+        if (nodeId instanceof Object) {
+          return Object.assign(nodeId, { description: description });
+        }
+        return {
+          get: async () => nodeId,
+          exists: async () => {
+            eventEmitter.emit('success', 'Exists');
+            return true;
+          },
+          text: async () => {
+            const result = await runtimeCallFunctionOn(
+              getText,
+              null,
+              { nodeId },
+            );
+            return result.result.value;
+          },
+          description,
+        };
+      });
+      return elementWrappers;
     } catch (e) {
-      return [];
+      return [
```

Comment:
We cannot return an array with a dummy object here. The `.elements` api will not work properly if no element is found.
Eg. if element does not exist, `.elements.length` is not equal to 0.